### ISSUE TITLE
Alpine Support

### ISSUE THREAD

**Issue by cirolosapio** [ynqa/sig#6]
is there a way to install on alpine ? 

i've tryed this steps

```sh
wget https://github.com/ynqa/sig/releases/download/v0.1.2/sigrs-x86_64-unknown-linux-gnu.tar.xz \
  && tar -xf sigrs-x86_64-unknown-linux-gnu.tar.xz \
  && mv sigrs-x86_64-unknown-linux-gnu/sig /usr/local/bin/sig
```

but it seems that some libs are missing

```
ldd sig         
/lib/ld-musl-x86_64.so.1: cannot load sig: No such file or directory
```

**Comment by ynqa** [comment]
@cirolosapio I just released [v0.1.3](https://github.com/ynqa/sig/releases/tag/v0.1.3) for MUSL Linux. Please confirm~

My attempts:
```bash
docker run -it --entrypoint=sh alpine

# in container
$ wget https://github.com/ynqa/sig/releases/download/v0.1.3/sigrs-x86_64-unknown-linux-musl.tar.xz \
  && tar -xf sigrs-x86_64-unknown-linux-musl.tar.xz \
  && mv sigrs-x86_64-unknown-linux-musl/sig /usr/local/bin/sig

$ sig --version
sig 0.1.3
```

**Comment by ynqa** [comment]
I will close this issue, but please feel free to reopen it if anything comes up :)
